src/image.js

```diff
@@ -101,7 +101,7 @@ image.stripEXIF = async function (path) {
 		const sharp = requireSharp();
 		await sharp(buffer, { failOnError: true }).rotate().toFile(path);
 	} catch (err) {
-		winston.error(err);
+		winston.error(err.stack);
```

Comment:
Intentional? I am not actually sure whether we can pass `err` to winston, I always just pass `err.message` but I vaguely recall that perhaps we updated it to allow handling the error object...

Reply:
I can't remember the exact details but there are some issues with winston3.0 not logging error objects properly. More info https://github.com/winstonjs/winston/issues/1338#issuecomment-431053451